lib/pbench/cli/server/user_create.py

```diff
@@ -0,0 +1,90 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
+
+_NAME_ = "pbench-user-create"
+
+
+class CreateUser:
```

Comment:
Ha; it's a little jarring that module `user_create.py` defines a class called `CreateUser`, but that's a nit at this point.